projects/plugins/beta/changelog/add-beta-breadcrumb-menu

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
I'd say "patch", not "minor". It's just a UI tweak, not really new functionality.

Reply:
https://github.com/Automattic/jetpack/pull/20394/commits/939379f417a16b1c61eb45a7c0c809bf5a5cdc96